geometry_msgs/Pose wish_pose
---
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages
